app/lib/katello/http_resource.rb

```diff
@@ -19,7 +19,7 @@ def initialize(params)
     include Katello::Concerns::FilterSensitiveData
 
     class_attribute :consumer_secret, :consumer_key, :ca_cert_file, :prefix, :site, :default_headers,
-                    :ssl_client_cert, :ssl_client_key
+                    :ssl_client_cert, :ssl_client_key, :ssl_ca_file
```

Comment:
I now see there already is `ca_cert_file` for this. I don't know what other classes can be passes into `pulp3_ssl_configuration` so I'm looking for guidance here.

Reply:
I included a commit to rename it for consistency with other tools and naming for the other `ssl_` attrs.